app/controllers/api/v1/lesson_parts_controller.rb

```diff
@@ -29,6 +29,7 @@ def create
     if lesson_part.save
       render(json: serialize(lesson_part).to_json, status: :created)
     else
+      byebug
```

Comment:
:bug:

Reply:
haha! Yeah this was still wip as I was doing both at the same time! I'll fix this one up and fix the remaining broken test 👍